### ISSUE TITLE
[Bug]: ABS result log

### ISSUE THREAD

**Issue by dengn** [matrixorigin/matrixone#3317]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```markdown
- Version or commit-id (e.g. v0.1.0 or 8b23a93): 6dedae20180c9e7ad4159ef0c7ad0f8e8c358dee
- Hardware parameters: MAC M1 ARM 16GB
- OS type: OSX
- Others:
```

### Actual Behavior

solved:
==================
[SQL STATEMENT]: select abs(cast(-2 as unsigned)), abs(18446744073709551614), abs(-2);
[EXPECT RESULT]:
abs(cast(-2 as unsigned))	abs(18446744073709551614)	abs(-2)
18446744073709551614	18446744073709551614	2
[ACTUAL RESULT]:
abs(cast(-2 as unsigned unsigned))	abs(18446744073709551614)	abs(-2)
18446744073709551614	2	2

unsolved:
=================
CREATE TABLE t1(c0 INTEGER, c1 INTEGER, c2 INTEGER);
CREATE TABLE t2(c0 INTEGER, c1 INTEGER, c2 INTEGER);
INSERT INTO t1 select abs(-341.741), abs(-234141113), abs(-141241);
INSERT INTO t2 select abs(-341.5612), abs(-23413), abs(-14141);
select * from t1;
select * from t2;

[SQL STATEMENT]: select * from t1;
[EXPECT RESULT]:
c0	c1	c2
342	234141113	141241
[ACTUAL RESULT]:
c0	c1	c2
341	234141113	141241

[SQL STATEMENT]: select * from t2;
[EXPECT RESULT]:
c0	c1	c2
342	23413	14141
[ACTUAL RESULT]:
c0	c1	c2
341	23413	14141

### Expected Behavior

CREATE TABLE t1(c0 INTEGER, c1 INTEGER, c2 INTEGER);
CREATE TABLE t2(c0 INTEGER, c1 INTEGER, c2 INTEGER);
INSERT INTO t1 select abs(-341.741), abs(-234141113), abs(-141241);
INSERT INTO t2 select abs(-341.5612), abs(-23413), abs(-14141);
select * from t1;
c0	c1	c2
342	234141113	141241
select * from t2;
c0	c1	c2
342	23413	14141

**Comment by fengttt** [comment]
my:dump@127.0.0.1:6001/foo=> select abs(cast(-2 as unsigned));
(0 rows)

my:dump@127.0.0.1:6001/foo=>

**Comment by daviszhen** [comment]
refresh error message

**Comment by dongdongyang33** [comment]
>CREATE TABLE t1(c0 INTEGER, c1 INTEGER, c2 INTEGER);
CREATE TABLE t2(c0 INTEGER, c1 INTEGER, c2 INTEGER);
INSERT INTO t1 select abs(-341.741), abs(-234141113), abs(-141241);
INSERT INTO t2 select abs(-341.5612), abs(-23413), abs(-14141);

>[SQL STATEMENT]: select * from t1;
[EXPECT RESULT]:
c0 c1 c2
342 234141113 141241
[ACTUAL RESULT]:
c0 c1 c2
341 234141113 141241

>[SQL STATEMENT]: select * from t2;
[EXPECT RESULT]:
c0 c1 c2
342 23413 14141
[ACTUAL RESULT]:
c0 c1 c2
341 23413 14141

This case will be fixed by [#3367](https://github.com/matrixorigin/matrixone/issues/3367)
